mpf/devices/multiball.py

```diff
@@ -123,7 +123,7 @@ def start(self):
         if self.balls_added_live - balls_added > 0:
             self.source_playfield.add_ball(balls=self.balls_added_live - balls_added)
 
-        if not self.config['shoot_again']:
+        if not self.config['shoot_again'].evaluate([]):
```

Comment:
I would only call evaluate once and store the value in a variable. It could chamge between calls (not likely here but possible).